lib/ansible/plugins/strategy/__init__.py

```diff
@@ -55,6 +55,12 @@
 
 __all__ = ['StrategyBase']
 
+# This list can be an exact match, or start of string bound
+# does not accept regex
+ALWAYS_DELEGATE_FACTS = frozenset((
```

Comment:
Name bikeshed- suggest `ALWAYS_DELEGATE_FACT_PREFIXES` for clarity

Reply:
done.